### DISCUSSION TITLE
Warum erfolgen keine Reaktionen mehr?

### DISCUSSION THREAD

**Discussion by JPRuehmann** [tine-groupware/tine#93] [General]
Warum werden nur noch Fehlerhafte Versionen veröffentlicht (wenn überhaupt) und auf Meldungen dan nicht mehr reagiert?
Statt dessen Beweihräuchert mann sich selbst in Podcasts zur neuen tollen Version.

**Comment by pschuele** [comment]
hallo,
das ist kein besonders konstruktiver beitrag - eigentlich möchte ich auf diesem niveau nicht kommunizieren.

heute kommt übrigens eine neue version raus.

und es kann leider immer etwas dauern, bis ich/wir die zeit finde/n, auf issues/fragen zu reagieren. da möchte ich um etwas geduld bitten.

wünsche schöne feiertage

**Comment by JPRuehmann** [comment]
fast drei Wochen ist aber doch ein bischen viel, zumal im Internet.
Zumal es ja nicht so ist das ich zuerst diesen Thread losgetreten habe.
